vad_transcriber/audioTranscript_gui.py

```diff
@@ -266,15 +285,84 @@ def transcriptionStart_on_click(self):
 
         # Execute
         self.threadpool.start(worker)
-
+   
+     
+    @pyqtSlot()
+    def CheckConvert_SR_on_click(self):  
```

Comment:
@pratibhaDSLearner I still think you should fix that in `wavTranscriber` directly, because right now you only fix the problem on the GUI and not on the CLI interface :)

Reply:
sure , i will do that ,thank you for your suggestion